BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -38,6 +38,7 @@ public struct AccessibilityElement: Element {
     public var identifier: String?
     public var traits: Set<Trait>
     public var accessibilityFrameSize: CGSize?
+    public var accessibilityFrameCornerStyle: Box.CornerStyle
```

Comment:
You'll likely always want to just match a box's corner style, so using the same value.